src/elements/speaker-details.html

```diff
@@ -0,0 +1,224 @@
+            sizing="cover"
+            lazy-load preload fade
+          ></plastic-image>
+          <h2 class="name" flex>[[speaker.name]]</h2>
+          <paper-icon-button
+            icon="hoverboard:close"
+            on-tap="close"
+            hidden$="[[viewport.isPhone]]"
+          ></paper-icon-button>
+        </app-toolbar>
+      </app-header>
+
+      <div class="content">
+        <ul class="content-list">
+          <li class="content-list-item" layout horizontal>
+            <iron-icon class="content-list-icon" icon="hoverboard:location"></iron-icon>
+            <div class="content-list-details" flex>[[speaker.country]]</div>
+          </li>
+          <li class="content-list-item" layout horizontal>
+            <iron-icon class="content-list-icon" icon="hoverboard:work"></iron-icon>
+            <div class="content-list-details" flex>[[speaker.title]], [[speaker.company]]</div>
+          </li>
+          <li class="content-list-item" hidden$="[[isEmpty(speaker.badges)]]" layout horizontal>
+            <iron-icon class="content-list-icon" icon="hoverboard:achievement"></iron-icon>
+            <div flex>
+              <template is="dom-repeat" items="[[speaker.badges]]" as="badge">
+                <div class="content-list-details">
+                  <a
+                    class$="badge"
+                    href$="[[badge.link]]"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    title$="[[badge.description]]"
+                  >
+                    [[badge.description]]
+                  </a>
+                </div>
+              </template>
+            </div>
+          </li>
+          <li class="content-list-item" layout horizontal>
+            <iron-icon class="content-list-icon" icon="hoverboard:document"></iron-icon>
+            <div class="content-list-details" flex>[[speaker.bio]]</div>
+          </li>
+        </ul>
+
+        <div class="contacts">
+          <template is="dom-repeat" items="[[speaker.socials]]" as="social">
+            <a href="[[social.link]]" target="_blank" rel="noopener noreferrer">
+              <paper-icon-button
+                class="social-icon"
+                icon="hoverboard:{{social.icon}}"
+              ></paper-icon-button>
+            </a>
+          </template>
+        </div>
+      </div>
+    </app-header-layout>
+
+  </template>
+
+  <script>
+
+    class SpeakerDetails extends UtilsFunctions(ReduxMixin(Polymer.mixinBehaviors([Polymer.IronOverlayBehavior], Polymer.Element))) {
+      static get is() {
+        return 'speaker-details';
+      }
+
+      static get properties() {
+        return {
+          speaker: {
+            type: Object
+          },
+          viewport: {
+            type: Object,
+            statePath: 'ui.viewport'
+          }
+        }
+      }
+
+      static get observers() {
+        return [
+          '_handleClose(opened)'
```

Comment:
`opened` is defined in `iron-overlay-behavior.html`. We pass this value from `hoverboard-app.html` and it's connected to redux store